src/Lab/ImGuiVulkan/ImGuiVulkanApplication.cs

```diff
@@ -0,0 +1,1211 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Input;
+using Silk.NET.Maths;
+using Silk.NET.Vulkan;
+using Silk.NET.Vulkan.Extensions.EXT;
+using Silk.NET.Vulkan.Extensions.ImGui;
+using Silk.NET.Vulkan.Extensions.KHR;
+using Silk.NET.Windowing;
+using Image = Silk.NET.Vulkan.Image;
+
+namespace ImGuiVulkan
+{
+	public class ImGuiVulkanApplication
+	{
+		public const bool EnableValidationLayers = true;
```

Comment:
After a reformat this file will be fine, we're not as fussy over lab applications so long as they work.